pages/portfolio.html

```diff
@@ -1,12 +1,79 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
+  <meta charset="UTF-8"/>
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Portfolio</title>
+  <link href="../styles/style.css" rel="stylesheet" />
 </head>
 <body>
-  
+  <section class="container">
+  <header class="cell-1">
+    <nav>
+      <ul id="main-menu">
+        <li>
+          <a class="button" href="index.html">Home</a>
+        </li>
+        <li>
+          <a class="button" href="about.html">About</a>
+        </li>
+        <li>
+          <a class="button" href="portfolio.html">Portfolio</a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <main class="cell-6">
+    <section id="viewing-party" class="projects">
+      <section id="viewing-party--image" class="image-size">
+        <a href="https://github.com/mell2022/viewing-party" target="_blank" rel="noopener noreferrer">
+        <img class="port-image" src="../assets/corina-rainer-P2wLo_PzHjU-unsplash.jpg" alt="popcorn in red bags">
+        </a>
+      </section>
+      Viewing Party
```

Comment:
Careful here! Text shouldn't be displayed by itself. It should be inside a heading, paragraph, etc.